docs/installation/install.md

```diff
@@ -3,6 +3,12 @@ title: Installation
 page_id: install
 ---
 
+## Requirements
+
+This project requires Python 3.7+.
+For Ubuntu it means that Ubuntu 20.04+ is supported unless a more recent version of Python is installed.
```

Comment:
This sentence is a bit too easy to misinterpreted. Perhaps just remove so that people can find the version for them self?

Reply:
Perhaps better to say. Ubuntu 20.04 has this installed for standard but for 18.04 this will need to be upgraded? and then also say that anything below 18.04 is not supported

Reply:
Removed. Ubuntu 16.04 is not supported for free by Canonical anymore so I just ignored it (it was already out of support from us).